src/components/page-active-component/component-replica-list.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import ReplicaSummaryModel from '../../models/replica-summary';
+import React from 'react';
+import { Typography } from '@equinor/eds-core-react';
+import { getReplicaUrl } from '../../utils/routing';
+import { ReplicaList } from '../replica-list';
+
+const replicatUrlFuncFactory =
+  (appName, envName, componentName) => (replicaName) =>
+    getReplicaUrl(appName, envName, componentName, replicaName);
+
+export const ComponentReplicaList = ({
+  appName,
+  envName,
+  componentName,
+  replicaList,
+}) => {
+  return (
+    <React.Fragment>
```

Comment:
Omit the optional React.Fragment inside of empty tags
```suggestion
    <>
```

Reply:
fixed